crawlers/mooncrawl/mooncrawl/leaderboards_generator/utils.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+import json
+import logging
+import os
+import time
+from typing import Any, Dict, Optional
+
+
+import requests  # type: ignore
+
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+def get_results_for_moonstream_query(
```

Comment:
This handles POST request, I sugest to use name: `update_results...` or so

Reply:
Not sure about this, anyway better to clarify at least in function description.

Reply:
Added in description.